src/pages/younseop/Main/Components/Aside.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Story from './Story';
+import Recommend from './Recommend';
+import Footer from './Footer';
+
+const Aside = () => {
+  return (
+    <aside className="mainRight">
+      <div className="asideWrapper">
+        <div className="profile">
+          <img
+            src="images/younseop/post-user.jpg"
+            alt=""
+            className="imgUserPhoto"
```

Comment:
```suggestion
          <img
            src="images/younseop/post-user.jpg"
            alt="imgUserPhoto"
            className="imgUserPhoto"
```

alt 값에는 시각장애인들도 인식할 수 있게 꼭 무슨 내용인지 명시해주셔 합니다!